source/System.cpp

```diff
@@ -99,7 +99,7 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 
 	// For the following keys, if this data node defines a new value for that
 	// key, the old values should be cleared (unless using the "add" keyword).
-	set<string> shouldOverwrite = {"asteroids", "attributes", "belt", "fleet", "link", "object", "hazard"};
+	set<string> shouldOverwrite = {"asteroids", "attributes", "belt", "fleet", "link", "random link", "object", "hazard"};
```

Comment:
Can you move `hazard` to between `fleet` and `link` and put `random link` at the end so its alphabetised?